source/slang/slang-check-shader.cpp

```diff
@@ -1513,7 +1513,7 @@ namespace Slang
         List<Expr*> globalSpecializationArgs;
         parseSpecializationArgStrings(
             endToEndReq,
-            endToEndReq->globalSpecializationArgStrings,
+            endToEndReq->m_globalSpecializationArgStrings,
```

Comment:
Grumble: I'm not a fan of introducing a "cleanup" like this into a change that isn't really related to coding convention or cleanup work. I'm still not 100% bought in on the idea that the `m_` prefix should be something that our convention *requires* as opposed to something it merely *allows*.

Reply:
It may be worth saying that I did not add the m_ without cause - perhaps I should have made more explicit what the cause was. With the sp functions we have asInternal produce variable called typically req, and then we access things via req->. With the move to the functionality to methods on EndToEndRequest in several methods the parameter name was the same as the member variable name - and confusion bugs ensued. 

That did not require *all* of the variables to be m_ prefixed it's true, but it seemed entirely arbitrary to have some have it and others not. There was also a variable with the other style.

Reply:
That makes more sense given the context. Thanks.

Honestly, if we could just agree on a `_` prefix for both internal functions and internal data, I'd be 100% on board with a sweeping cleanup of the code to change this stuff. It's that little `m` that ties me up in knots. :)